sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,68 @@
 Version history
 ===============
 
+6.10.0 (2022 May)
+-----------------
+
+New file formats:
+
+* Slidebook 7
+   - added support for reading SlideBook Format 7 from SlideBook 2021
+     This functionality was implemented and contributed by Intelligent Imaging Innovations
+
+File format fixes and improvements:
+
+* CellWorX / MetaXpress
+   - channel metadata will now be preserved for all channels by reading from each file
+
+* Cellomics
+   - marked the private Pattern field as transient to fix memo file generation with JDK17
+
+* DeltaVision
+   - fixed detection of incorrect XY tile counts
+
+* Hitachi S-4800
+   - enhanced format recognition and identification of datasets
+   - corrected the units for physical sizes to nanometers instead of micrometers
+
+* Nikon NIS-Elements ND2
+   - improved metadata parsing, especially for objective data, timestamps, and exposure times
+
+* PerkinElmer Operetta
+   - improved checks for invalid TIFFs and supplemental metadata files
+   - updated image names to make well names more readable
+
+* TIFF (Tagged Image File Format)
+   - fixed incorrect SampleValue metadata values
+
+* Vectra QPTIFF
+   - reader has been updated to support schema version 4 
+     Funded by a partnership between Glencoe Software and Akoya Biosciences.
```

Comment:
This does not format well. 
![Screenshot 2022-05-31 at 11 07 49](https://user-images.githubusercontent.com/2478303/171149739-dac4dd07-5741-4c20-a7af-55267a4c0743.png)

Reply:
```suggestion
    - Note:  Funded by a partnership between Glencoe Software and Akoya Biosciences.
```

Maybe like this ?